### PR TITLE
Update gpsbabel

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- shapelib: Add at 1.6.0
- gpsbabel: Update to 1.9.0 (Resolves https://github.com/getsolus/packages/issues/2284)

### CODE REVIEW

**Comment by androidnisse** [comment]
Resolves https://github.com/getsolus/packages/issues/2284

File: packages/s/shapelib/pspec_x86_64.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        <Summary xml:lang="en">Development files for shapelib</Summary>
+        <Description xml:lang="en">Simple C API for reading and writing ESRI Shapefiles
+</Description>
+        <PartOf>programming.devel</PartOf>
+        <RuntimeDependencies>
+            <Dependency release="1">shapelib</Dependency>
+        </RuntimeDependencies>
+        <Files>
+            <Path fileType="header">/usr/include/shapefil.h</Path>
+            <Path fileType="library">/usr/lib64/libshp.a</Path>
```
Comment (davidjharder) [inline packages/s/shapelib/pspec_x86_64.xml:59]: Don't need the static libs

I would say we can ditch the `-devel` package and just pattern everything into `shapelib`

File: packages/s/shapelib/monitoring.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+releases:
+  id: 4804
+  rss: ~
```
Comment (davidjharder) [inline packages/s/shapelib/monitoring.yml:3]: Small nit: There is a github repo we can use: https://github.com/OSGeo/shapelib for a feed

**Review by davidjharder — CHANGES_REQUESTED** [review]
Some small things. Great work on this.

**Review by davidjharder — APPROVED** [review]
LGTM, thanks!

**Pull request merged**